ADMVEDU/ViewControllers/ViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ViewController.swift
+//  ADMVEDU
+//
+//  Created by Bobr, Andrey on 4.05.21.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    private let searchController = UISearchController(searchResultsController: nil)
+    private var tableView = UITableView()
+    
+    private let service = MediaService()
+    private var requestOptions = RequestOptions()
+    private var results = [ResultData]()
```

Comment:
var results: [ResultData] = [] is preferable
https://github.com/raywenderlich/swift-style-guide/blob/master/README.markdown